akirataguchi115/lukuvinkkikirjasto
# Rename some packages

## Description
I renamed some packages, so that the language of our code is more cohesive.

## How to test:
Test that the project still works as it should with `./gradlew run`. Check that the new names are fine.
